src/util/svgUtils/types.ts

```diff
@@ -0,0 +1,99 @@
+export type SVGElementAttributeTypes = {
+  '*': ReadonlyArray<string>;
```

Comment:
what is `*`

Reply:
It is called the universal key. i kept it as it is, i wasn't use we use it. ps://github.com/wooorm/svg-element-attributes/blob/main/index.js#L7

Reply:
we could probably omit this one

Reply:
done